### DISCUSSION TITLE
WordPress support

### DISCUSSION THREAD

**Discussion by uhlhosting** [WeblateOrg/weblate#7461] [Ideas]
Hi,

Any plans or ETA for wordpress support? 

This would be amaZing to have since on the market there or very few options or none open source based.

**Comment by nijel** [comment]
There is currently no ETA. There was some discussion in https://github.com/WeblateOrg/weblate/issues/1066. Meanwhile, several third-party integrations were developed, see https://docs.weblate.org/en/latest/devel/thirdparty.html
